ts.config.js

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "module": "commonjs",
+    "target": "es6",
+    "allowJs": true,
+    "rootDir": "src",
+    "outDir": "dist",
+    "sourceMap": true,
+    "strict": false,
```

Comment:
Eventually we will want to use strict mode. Fine to leave it off for now.

Reply:
We'll need to leave it off until we're fully refactored right?